src/io/ioexternalbus.h

```diff
@@ -37,13 +40,25 @@ class IOExternalBus : public IOBase {
     virtual void ioWrite(AInt offset, VInt value, unsigned size) override;
 
 protected:
-    virtual void parameterChanged(unsigned) override;
+    virtual void parameterChanged(unsigned) override{/* no parameters */};
+
+private slots:
+    void connectButtonTriggered();
 
 private:
+    uint32_t ByteSize;
+    int32_t send_cmd(const uint32_t cmd, const uint32_t payload_size = 0, const QByteArray* payload = nullptr);
```

Comment:
Can we use `const QByteArray& payload = {}` instead?

Reply:
Yes, of course.